packages/varlet-ui/types/actionSheet.d.ts

```diff
@@ -23,11 +23,11 @@ export interface ActionSheetProps extends BasicAttributes {
 
 export interface ActionItem {
   name: string
-  color: string
-  icon: string
-  iconSize: string | number
-  className: string
-  disabled: boolean
+  color?: string
+  icon?: string
+  iconSize?: string | number
+  className?: string
+  disabled?: boolean
```

Comment:
LGTM. From [action-sheet.vue](https://github.com/varletjs/varlet/blob/dev/packages/varlet-ui/src/action-sheet/ActionSheet.vue#L27-L42). Type declaration to be optional is better.